websocket-sharp.clone/Ext.cs

```diff
@@ -1221,7 +1046,7 @@ internal static byte[] ToByteArray(this Stream stream)
             using (var output = new MemoryStream())
             {
                 stream.Position = 0;
-                stream.CopyTo(output);
+                await stream.CopyToAsync(output).ConfigureAwait(false);
                 output.Flush();
```

Comment:
Doesn't really matter here since we are using `MemoryStream` but for consistency reasons we could use `FlushAsync`